.github/workflows/deploy-wiki.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy wiki
+
+
+on:
+  push:
+    paths:
+      # Trigger only when wiki directory changes
+      - 'wiki/**'
+    branches:
+      # And only on master branch
+      - master
+
+jobs:
+  deploy-wiki:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # Fetch merge commit and last commit from pull request
+        fetch-depth: 2
+
+    - name: Set environment
+      run: |
+        # Skip if HEAD is a merge commit
+        if [[ $(git log -n 1 --pretty=format:%s) == "Merge Pull Request"* ]]; then
+          echo "Skipping Merge commit"
+          CMD='git log -n 1 --skip=1'
+        else
+          echo "Using current commit"
+          CMD='git log -n 1'
+        fi
+        # Set the author name, author email, and message from the pull request
+        echo "::set-env name=ACTION_MAIL::$(eval $CMD --pretty=format:%ae)"
+        echo "::set-env name=ACTION_NAME::$(eval $CMD --pretty=format:%an)"
+        echo "::set-env name=WIKI_PUSH_MESSAGE::$(eval $CMD --pretty=format:%s)"
```

Comment:
@n1ru4l, This checks if HEAD is a merge commit and skips back one commit if it is. This should handle squashed commits the right way.

Reply:
It seems a bit hacky but should do the job, so let's keep it like that. We can come up with a better solution later 😊